test/functionalTest/cases/1592_query_with_q_op_greater_or_equal/query_with_all_q_ops_without_right_hand_side.test

```diff
@@ -30,27 +30,35 @@ brokerStart CB
 --SHELL--
```

Comment:
(Discussed at skype) Add some .test to queck attribute existence and not-existence in query operation.